services/gateway/apis/http-version3/swagger/parameters/dynamicReward.json

```diff
@@ -0,0 +1,9 @@
+{
+    "blockHeight": {
```

Comment:
```suggestion
    "rewardBlockHeight": {
```

Reply:
The param is not dependant on reward module and can reused whenever we need to query for specific block height.

Reply:
Then this shouldn't be defined within the param file specific to `dynamicReward`. Please move it to the appropriate file.

Reply:
moved to `common.json`